### PR TITLE
Extract migration for author page with workaround

### PR DESCRIPTION
**Pull request by tricknotes**

### CODE REVIEW

File: app/models/participant.rb [hunk 1]
```diff
@@ -6,4 +6,18 @@ class Participant < ActiveRecord::Base
...
+
+  def description
+    respond_to?(:summary) ? summary : super
+  end
+
+  def description=(description)
+    if respond_to?(:summary=)
+      self.summary = description
+    else
+      write_attribute(:description, description)
```
Comment (tricknotes) [inline app/models/participant.rb]: haha

Comment (mtsmfm) [inline app/models/participant.rb]: How about `super` ?

Comment (tricknotes) [inline app/models/participant.rb]: seems good 👍

**Comment by mtsmfm** [comment]
Awesome work!

**Pull request merged**